.github/workflows/ci.yml

```
@@ -0,0 +1,31 @@
+on:
+  push:
+    branches: [ master ]
+  pull_request:
```

I would have expected this:
```suggestion
  pull_request: [ master ]
```

But I think that only limits the builds being run. It shouldn't cause a build to be run that's not run now?